src/backend/base/langflow/services/auth/utils.py

```diff
@@ -164,17 +165,17 @@ async def get_current_user_for_websocket(
     if token:
         return await get_current_user_by_jwt(token, db)
     if api_key:
-        return await asyncio.to_thread(api_key_security, api_key, query_param, db)
+        return await asyncio.to_thread(api_key_security, api_key, query_param)
     return None
 
 
-def get_current_active_user(current_user: Annotated[User, Depends(get_current_user)]):
+async def get_current_active_user(current_user: Annotated[User, Depends(get_current_user)]):
```

Comment:
`get_current_user` is async so it's better to make `get_current_active_user` async (otherwise fastapi has to create a new event loop to run it).